plugins/event_source/alertmanager.py

```diff
@@ -74,6 +109,12 @@ def clean_host(host):
 async def main(queue: asyncio.Queue, args: Dict[str, Any]):
     app = web.Application()
     app["queue"] = queue
+    app["data_host_path"] = args.get("data_host_path") or "labels.instance"
+    app["data_path_separator"] = args.get("data_path_separator") or "."
+    app["data_alerts_path"] = args.get("data_alerts_path")
+    if app["data_alerts_path"] is None:
+        app["data_alerts_path"] = "alerts"
+    app["skip_original_data"] = args.get("skip_original_data")
```

Comment:
Suggest to use `args.get("skip_original_data", False)`, feels it's more clear if default is `false`